### PR TITLE
Bug 1729524 Add search_terms adm_weekly_aggregrates

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1729524

There will be a separate PR with a new DAG for exporting this data.

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+SELECT
+  search_query AS query,
+  block_id,
+  position,
```
Comment (rebecca-burwei) [inline sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql:4]: There are some strange values for position (i.e., values much greater than 10). If we don't clean this up, we'll likely get questions about it from adM via Aaron. @teonbrooks , have you found reasons why there can be urlbar positions > 10? 

My preference would be to send adM the raw data (unless they explicitly asked for something else), but have some answers on hand for when they ask

**Review by rebecca-burwei — COMMENTED** [review]
For sponsored tiles, legal said that we should not send advertiser info joined to tile position. I'm not sure if this applies to Firefox Suggest, but here, block_id would encode advertiser info for adM.

File: sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql (hunk 1 above)
Comment (jklukas) [reply sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql:4]: Per discussion with AKim, we will remove position.

Comment (jklukas) [inline sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql:4]:
```suggestion
```

**Review by rebecca-burwei — APPROVED** [review]
lgtm

File: sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+FROM
+  search_terms.suggest_impression_sanitized
+WHERE
+  DATE(submission_timestamp)
+  BETWEEN @submission_date
+  AND DATE_SUB(@submission_date, INTERVAL 6 DAY)
+GROUP BY
+  query,
+  block_id,
+  position
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql:15]:
```suggestion
```

Comment (jklukas) [inline sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql:15]:
```suggestion
  block_id
```

**Pull request merged**